Makefile

```
@@ -0,0 +1,5 @@
+.PHONY: check test-unit
+
+test-unit:
+	cd tests && PYTHONDONTWRITEBYTECODE="nono" python3 -m pytest --showlocals --verbose test_*
```

```suggestion
	cd tests && PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --showlocals --verbose test_*
```
I searched but didn't find what `PYTHONDONTWRITEBYTECODE="nono"` does. Can you please explain?

---

If `PYTHONDONTWRITEBYTECODE` is specified, it prevents Python to cache its bytecode into `__pycache__` folder. https://docs.python.org/3/using/cmdline.html#envvar-PYTHONDONTWRITEBYTECODE

---

Oh, thx, so there can be any non-empty string. In that case, I would still suggest writing "1" instead of "nono". Except, if this was some tradition, which I do not know about :)

---

> If `PYTHONDONTWRITEBYTECODE` is specified, it prevents Python to cache its bytecode into `__pycache__` folder. https://docs.python.org/3/using/cmdline.html#envvar-PYTHONDONTWRITEBYTECODE

@frenzymadness What do you prefer instead of that? 'tutununu'

---

I don't have a preference here. Whatever works for me.